neonvm/apis/neonvm/v1/virtualmachine_types.go

```diff
@@ -390,6 +390,16 @@ const (
 	VmMigrating VmPhase = "Migrating"
 )
 
+// IsAlive returns whether the guest in the VM is expected to be running
+func (p VmPhase) IsAlive() bool {
+	switch p {
+	case VmRunning, VmMigrating:
```

Comment:
need add there `VmPreMigrating` as well I think. It's short time phase (about 1 second usually) but anyway.

Reply:
That'll be required with #256, right? (`VmPreMigrating` isn't on main yet)